app/helpers/application_helper.rb

```diff
@@ -1219,7 +1219,10 @@ def allowed_filter_db?(db)
   end
 
   def miq_tab_header(id, active = nil, options = {}, &block)
-    content_tag(:li, :class => "#{options[:class]} #{active == id ? 'active' : ''}", :id => "#{id}_tab") do
+    content_tag(:li,
+                :class     => "#{options[:class]} #{active == id ? 'active' : ''}",
+                :id        => "#{id}_tab",
+                'ng-click' => "changeAuthTab('#{id}');") do
```

Comment:
Fixed in https://github.com/ManageIQ/manageiq-ui-classic/pull/68/commits/2b1da75e4d389b1212f3db1b2885016a047846ee